### PR TITLE
fix: performance issue

### PR DESCRIPTION
**Pull request by gxcsoccer**
image

### CODE REVIEW

File: src/common.js [hunk 1]
```diff
@@ -64,15 +64,15 @@ function setup(env) {
...
-      }
-
       // Disabled?
       if (!debug.enabled) {
         return;
       }
 
+      for (var _len = arguments.length, args = new Array(_len), _key = 0; _key < _len; _key++) {
+        args[_key] = arguments[_key];
+      }
```
Comment (gxcsoccer) [inline src/common.js]: should not run if debug is disabled

**Comment by fengmk2** [comment]
LGTM

**Comment by Qix-** [comment]
Does this exist in master, too?

**Comment by gxcsoccer** [comment]
> Does this exist in master, too?

@Qix- master is ok

**Comment by Qix-** [comment]
Released as `debug@3.2.6`.

**Pull request merged**